packages/amplify-provider-awscloudformation/src/initializer.ts

```diff
@@ -53,6 +55,40 @@ export async function run(context) {
     const authRoleName = `${stackName}-authRole`;
     const unauthRoleName = `${stackName}-unauthRole`;
 
+    const configuration = {
+      authRole: {
+        roleName: authRoleName,
+      },
+      unauthRole: {
+        roleName: unauthRoleName,
+      },
+    };
+
+    const noOverrideMsg = 'No override file found. To override, run amplify override auth';
+    try {
+      const backendDir = pathManager.getBackendDirPath();
+      const overrideFilePath = path.join(backendDir, 'awscloudformation', 'build', 'override.js');
+      const overrideCode: string = await fs.readFile(overrideFilePath, 'utf-8').catch(() => {
```

Comment:
1. It would be good to move this into a function like "transpileAmplifyOverridesCode()" and parameterize the path parameters .
2. It would be good to create a constant for the 5 sec timeouts.